packages/alchemy/src/client/lightAccountClient.ts

```diff
@@ -25,17 +25,21 @@ export type AlchemyLightAccountClientConfig<
     "account"
   >;
 
-export const createLightAccountAlchemyClient: <
+export async function createLightAccountAlchemyClient<
   TSigner extends SmartAccountSigner = SmartAccountSigner
 >(
   params: AlchemyLightAccountClientConfig<TSigner>
-) => Promise<
+): Promise<
   AlchemySmartAccountClient<
     CustomTransport,
     Chain | undefined,
     LightAccount<TSigner>
   >
```

Comment:
ah yea same comment here, I think this is missing the actions type. I wonder how this was working before 🤔

Reply:
added action typing